integration-cli/docker_utils.go

```diff
@@ -269,6 +269,14 @@ func (d *Daemon) Cmd(name string, arg ...string) (string, error) {
 	return string(b), err
 }
 
+func (d *Daemon) CmdWithArgs(daemonArgs []string, name string, arg ...string) (string, error) {
```

Comment:
@cpuguy83 I tried what you suggested and it complains having duplicate -H, I can write code to check that into Cmd but then what if the command itself has -h, etc. I can never make that deterministic.